pydis_site/apps/resources/resources/software_design_by_example.yaml

```
@@ -0,0 +1,25 @@
+name: Software Design by Example
+description: A tool-based introduction to Software Design with Python. This book teaches
+  design by explaining examples of small versions of familiar tools to show how
+  experienced software designers think. It introduces some fundamental ideas in computer
+  science that many self-taught programmers haven’t encountered.
+title_image: https://third-bit.com/sdxpy/logo.svg
```

```suggestion
icon_image: https://third-bit.com/sdxpy/logo.svg
```
I think this would cause the name to still be displayed next to the icon, which would be a bit clearer. I've not tried this out though.

There's also an `icon_size` to configure the size of the icon with this.

---

Thank you for reviewing this! I've changed it to `icon_image` and set `icon_size` to 50.